tests/test_sure_loss.py

```diff
@@ -65,7 +65,7 @@ def operator(x):
 
         loss_real = sure_loss_real(operator, x_real, y_pseudo_gt_real, complex_input=False)
         loss_complex = sure_loss_complex(operator, x_complex, y_pseudo_gt_complex, complex_input=True)
-        self.assertAlmostEqual(loss_real.item(), loss_complex.abs().item(), places=6)
+        self.assertAlmostEqual(loss_real.item(), loss_complex.abs().item(), places=5)
```

Comment:
Hi @borisfom, may I know the reason for this update here?

Reply:
I had CI failing intermittently because of very small diff with 6 places.